neurom/core/tests/test_soma.py

```diff
@@ -79,15 +79,6 @@
     [11, 22, 33, 44, 2, 1, 1]
 ]
 
-BIFURCATING_SOMA_PTS = [
```

Comment:
This was added to exclude branchings within the soma (which are not allowed in any other type of soma except swc). I don't know why this test was removed but it would be good to have one as currently the valid set of neurons contains files that have issues because they have bifurcating somata.